packages/apps/docs/src/github.td.ts

```diff
@@ -0,0 +1,172 @@
+interface IGitHubFile {
+  sha: string;
+  filename: string;
```

Comment:
the file extension should be `github.d.ts`, isn't it?
can't we use octokit types are other related types definition from octokit?